api/config/config_test.json

```diff
@@ -152,7 +152,10 @@
   },
   "reporting": {
     "dataset": "reporting_test",
-    "maxRowsPerInsert": 500,
+    "maxRowsPerInsert": 50000,
```

Comment:
These values were confirmed empirically.  (Note: I intentionally made `cohort` very large to force a failure at the larger batch size)